.github/workflows/publish_package_testpypi.yaml

```diff
@@ -0,0 +1,45 @@
+name: Publish to Test PyPI
+
+on:
```

Comment:
I would prefer if this was part of the `publish_package.yml` workflow and used a conditional to decide when the `prod` version gets published. It should look identical to the `docker.yml` workflow just about

Reply:
@allisonking lmk if there's a reason this wouldn't really work though, but it seems like it should

Reply:
ohh okay, yeah that makes sense I'll try it out!

Reply:
Okay I made the changes. I tried to test them on PR requests just to test it out here, but I think that breaks the check that I stole from `docker.yml` https://github.com/ethyca/fides/runs/7659389551?check_suite_focus=true since the event ref doesn't exist on PR's. So I'm guessing this will work since the docker one does, but I'm not 100% sure